packages/manager/modules/preloader/src/index.js

```diff
@@ -0,0 +1,43 @@
+import {
+  WELCOME_MESSAGES,
+  LOADING_MESSAGES,
+  SWITCH_PARAMETER,
+} from './constants';
+import buildTemplate from './template';
+
+const buildQueryParams = (search) => {
+  const queryParamsString =
+    search.indexOf('?') === 0 ? search.slice(1) : search;
+
+  return queryParamsString.split('&').reduce((params, pair) => {
+    const [name, value] = pair.split('=');
+    return {
+      ...params,
+      [name]: decodeURIComponent(value),
+    };
+  }, {});
+};
+
+export const attach = () => {
+  const queryParams = buildQueryParams(window.location.search);
+  const template = document.createElement('template');
+
+  if (!Object.keys(queryParams).includes(SWITCH_PARAMETER)) {
+    template.innerHTML = buildTemplate(false, WELCOME_MESSAGES);
+  } else {
+    template.innerHTML = buildTemplate(false, LOADING_MESSAGES);
+  }
+
```

Comment:
If the url contains `?m` we will change the messages from Welcome (`Welcome to the OVHcloud Control Panel`) to loading message (`Loading…`).
It could be used when we redirect customer form one application to another one.